src/views/pacbio/PacbioPoolIndex.vue

```diff
@@ -80,6 +80,11 @@
             :filter="filter"
           >
           </b-table>
+          <ul v-if="!row.item.run_suitability.valid">
+            <li v-for="(error, index) in row.item.run_suitability.formattedErrors" :key="index">
+              {{ error }}
+            </li>
+          </ul>
```

Comment:
Here we already have a more-details section, so adding the extra information is easy.